go.mod

```diff
@@ -100,6 +100,7 @@ require (
 	github.com/fxamacker/cbor/v2 v2.7.0
 	github.com/ghodss/yaml v1.0.0
 	github.com/gizak/termui/v3 v3.1.0
+	github.com/go-git/go-git/v5 v5.12.0
```

Comment:
`go-git` will be used for the `teleport` as well.

local mac build `tsh` increased about 200k (out of 106MB) after importing it.